.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
+#   directories:
+#   - $HOME/.glide
+
 go:
 - 1.7.1
 
 branches:
   only:
   - master
   - cluster-up
+  - /^issue-[0-9]+$/
```

Comment:
As long as we all work on forks generally then there is no need to restrict this at all. This was originally in here because I was the sole contributor & worked in the same repo so wanted to restrict builds to master branch only, otherwise PRs built both PR branch & PR. This doesn't apply now so I agree the restriction should be removed.